cmd/notary/main.go

```diff
@@ -53,3 +59,20 @@ func main() {
 	logger.Info("Shutting down server")
 	<-idleConnsClosed
 }
+
+func createEncryptionBackend(backendConfig config.BackendConfig, logger *zap.Logger) (encryption_backend.EncryptionBackend, error) {
+	switch backendConfig.Type {
+	case config.PKCS11:
+		backend, err := encryption_backend.NewPKCS11Backend(backendConfig.PKCS11.LibPath, backendConfig.PKCS11.Pin, *backendConfig.PKCS11.KeyID, logger)
+		if err != nil {
+			return nil, err
+		}
+		logger.Info("PKCS11 backend created")
```

Comment:
Sorry for being pedantic here, but we don't really "create" a backend, we connect to it. I'm pushing on these logs as they are the only way we currently inform users that they are using an HSM.